src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
```

Comment:
I see what you mean and I agree that it makes sense to check for the version when building.

But the problem is `find_package(Radare2 ...)` runs the FindRadare2.cmake script inside this repo under src/cmake and this script simply does not check the version at the current state.
So as long as this script is not adjusted to do so, the version should not be specified here, otherwise it makes people assume the version is checked which is not true.